recipes/nds2-client/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "nds2-client" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "34539f905428493c1a02a208fce0356dec5c8310bac90a83fa20d0d5ad1040d7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # fix bug in allowing user to set SASL path
+    - with-sasl.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python <3
```

Comment:
Are Python and SWIG needed for the client? If so, shouldn't they be host and run deps?

Reply:
@scopatz, these were unneeded and have been removed, see 1f30c18.